src/utils/get-config.ts

```diff
@@ -15,9 +14,9 @@ export interface Config {
 	/** Which directory to search for test projects. Defaults to `test-projects`. */
 	testProjectsDirectory: string;
 
-	/** An absolute path to the file Yarn commands should use for the `--mutex` flag. Defaults to a randomly
-	 * generated temporary file. */
-	yarnMutexFilePath: string;
+	/** A port Yarn commands should use for the `--mutex` flag. Defaults to a randomly
+	 * 31997. */
```

Comment:
This needs tweaking:

```suggestion
	/** The port Yarn commands should use for the `--mutex` flag. Defaults to 31997. */
```